include/odp/api/spec/classification.h

```diff
@@ -478,6 +536,37 @@ int odp_cos_with_l3_qos(odp_pktio_t pktio_in,
 			odp_cos_t cos_table[],
 			odp_bool_t l3_preference);
 
+/**
+ * Reset statistics for a CoS queue
+ *
+ * Reset all statistics counters to zero.
+ *
+ * @param cos          CoS handle
+ * @param queue        CoS event queue handle
+ *
+ * @retval  0 on success
+ * @retval <0 on failure
+ */
+int odp_cls_queue_stats_reset(odp_cos_t cos, odp_queue_t queue);
```

Comment:
Do we need the `cos` argument?

Reply:
In our HW, we would need cos argument as a Scheduler queues are independent entities and can be associated to multiple CoS objects.

I guess even odp_cls_cos_create() can take same odp_queue_t as input (num_queue = 1) multiple times.

Reply:
Ok